picard/ui/infodialog.py

```diff
@@ -98,7 +98,7 @@ def get_type_widget(self, type_text):
 class InfoDialog(PicardDialog):
 
     def __init__(self, obj, parent=None):
-        PicardDialog.__init__(self, parent)
+        sueper().__init__(parent)
```

Comment:
^^typo

Reply:
oops. sorry. will fix it in next commit.